Player Guides (#5)
FabricMC/fabric-docs: players/finding-mods.md

---

> title: Finding Trustworthy Mods

Reviewer: Great idea 👍

description: A guide on how to find Fabric mods using trustworthy sources.
---

# Finding Trustworthy Mods

Firstly, trust is subjective, and you should always use your own judgement when downloading mods. However, there are some things you can do to help you find trustworthy mods.

## 1. Use a source that is known to be trustworthy.

> The majority of authors publish their mods to [Modrinth](https://modrinth.com/mods?g=categories:%27fabric%27) and [CurseForge](https://www.curseforge.com/minecraft/search?page=1&pageSize=20&sortType=1&class=mc-mods&gameFlavorsIds=4), however some may choose to upload it on their personal websites, or on other platforms, such as a GitHub repository.

Reviewer 2: Do not mention personal websites/GitHub here - likely confusion

> These platforms have moderation teams in place that check that mods are what they say they are, and that they do not contain any malicious code. They also have a reporting system in place, so that if you do find a malicious mod, you can report it to the moderation team, and they will take action relatively quickly.

Reviewer 2: ```suggestion
These websites check that the mods are what they say they are, and that they do not contain malicious code. You can also report malicious mods on those websites, and they will take action relatively quickly.
```

## 2. Check with others!

If you are downloading a mod from a source that is not known to be trustworthy, you should check with others to see if they have downloaded the mod before from the location you're downloading from, and if they have had any issues with it.

If in doubt, you are welcome to ask in the [Fabric Discord](https://discord.gg/v6v4pMv) in the `#player-support` channel.

## 3. Avoid common malware sites!

::: info

> Malware sites may not be obvious to everyone, so if you are unsure, you should ask for opinions from others or avoid the site altogether and rely only on trusted sources, such as Modrinth and CurseForge.

Reviewer 2: ```suggestion
Malware sites may not be obvious to everyone. If you are unsure, you should ask for opinions from others or avoid the site altogether and rely only on trusted sources, such as Modrinth and CurseForge.
```

:::

> There are a lot of sites out there that claim to have mods for Minecraft, but are actually just malware sites. You should avoid these sites at all costs.

Reviewer 2: ```suggestion
There are a lot of websites that claim to have mods for Minecraft, but are actually just malware sites. You should avoid these sites at all costs.
```

- `10minecraft.ru`
- `creeper.gg`

A full comprehensive list of these sites can be found [here, courtesy of Stop Mod Reposts](https://stopmodreposts.org/).

> 

Reviewer: Remove this, this will only boost their SEO rankings IMO. And possibly give a false sense of security for the ones not listed.

SMR is no longer maintained, again a false sense of security.

> If you are unsure if a site is trustworthy, you can use a decent antivirus software, such as [Windows Defender](https://www.microsoft.com/en-us/windows/windows-defender) or [VirusTotal](https://www.virustotal.com/) to ensure that the site is not trying to trick you into downloading anything malicious onto your computer.

Reviewer 2: ```suggestion
You can use antivirus software and websites like [Windows Defender](https://www.microsoft.com/en-us/windows/windows-defender) or [VirusTotal](https://www.virustotal.com/) to check the downloaded mods. However, do not rely entirely on those methods, because they can be sometimes incorrect.
```
